src/Behat/Behat/Hook/Context/Attribute/HookAttributeReader.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Hook\Context\Attribute;
+
+use Behat\Behat\Context\Annotation\DocBlockHelper;
+use Behat\Behat\Context\Attribute\AttributeReader;
+use Behat\Hook\AfterFeature;
+use Behat\Hook\AfterScenario;
+use Behat\Hook\AfterStep;
+use Behat\Hook\BeforeFeature;
+use Behat\Hook\BeforeScenario;
+use Behat\Hook\BeforeStep;
+use Behat\Hook\Hook;
+use ReflectionMethod;
+
+class HookAttributeReader implements AttributeReader
+{
+    /**
+     * @var string[]
+     */
+    private static $classes = array(
+        AfterFeature::class => 'Behat\Behat\Hook\Call\AfterFeature',
```

Comment:
Becuase Behat is still riddles with FQCN strings like these and like to be consistent with existing code.

IMO it would be better to have a separate MR that replaces FQCN strings with `::class` "constants" everywhere in one go, rather than mix and match different syntaxes accross the project.

Basically the same reason I'm also using `array()` here rather than `[]`.

Reply:
but here, you used `::class` for keys. So you are not consistent with your own new code.

Reply:
It is however consistent with the [DefinitionAttributeReader](https://github.com/Behat/Behat/blob/master/src/Behat/Behat/Definition/Context/Attribute/DefinitionAttributeReader.php#L31-L35).

Reply:
I'm good with both approaches, we can address this issue later on - going all-in on `::class` syntax in a separate PR seems like a good move.